projects/ngrx.io/content/guide/entity/recipes/additional-state-properties.md

```diff
@@ -0,0 +1,84 @@
+# Additional Entity State Properties Update
+
+For beginners it is very confusing how to update the additional state properties using adapter. So we are going to see how to update the Additional Entity state properties. In the below example we are going to use the [Entity Adapter](https://ngrx.io/guide/entity/adapter) example.

```

Comment:
```suggestion
It's possible to add extra properties to a `State` extending from `EntityState`. These properties must be updated manually. Just like in a non-entity state, we can update the added properties in the reducer. This can be done with or without using the `@ngrx/entity` helper functions.

In the below example we are going to use the [Entity Adapter](https://ngrx.io/guide/entity/adapter) example.
```

Reply:
updated